### PR TITLE
8231231: The printing result is different from the case instruction

### PR DESCRIPTION
**Pull request by prsadhuk**
Please review a manual printing test failure where the instructions is not matching the printing result 
as the labels which are present in the test instruction frame is cut off in printed result due to margin not being accounted for, 
as the labels in the test instruction is fully left aligned starting from 0.
Fixed by adding some spaces before labels and 
for other labels drawn through drawChars, modified to draw the labels from coordinate 20 instead of 0 to allow labels being printed in the paper and not get cutoff by margin.
Also, the Print instruction is brought to top and 1st label is brought to bottom (SOUTH position in BorderLayout) as 1st label is in JScrollPane so to account for top margin cutoff, brought it lower so that those labels does not get cut off.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Testing

|     | Linux x32 | Linux x64 | Windows x64 | macOS x64 |
| --- | ----- | ----- | ----- | ----- |
| Build | ✔️ (1/1 passed) | ✔️ (5/5 passed) | ✔️ (2/2 passed) | ✔️ (2/2 passed) |
| Test (tier1) |    |  ✔️ (9/9 passed) | ✔️ (9/9 passed) | ✔️ (9/9 passed) |

### Issue
 * ⚠️ Failed to retrieve information on issue `8231231`.

### Reviewers
 * [Phil Race](https://openjdk.java.net/census#prr) (@prrace - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/780/head:pull/780`
`$ git checkout pull/780`

### CODE REVIEW

**Comment by mrserb** [comment]
Probably we should not use some hardcoded values but use some paper related insets? Don't we need to update the public int print(Graphics graphics) instead and scale the frame to the paper, or just translate the graphics passed to print() before passing it to this.paint()?

File: test/jdk/java/awt/print/bug8023392/bug8023392.java [hunk 1]
```diff
@@ -72,9 +73,9 @@ public void init() {
 
         public SimplePrint2() {
             setLayout(new BorderLayout());
-            label1 = new JLabel("2a) a b c d e" +
+            label1 = new JLabel("  2a) a b c d e" +
                     "                         ");
```
Comment (prrace) [inline test/jdk/java/awt/print/bug8023392/bug8023392.java:77]: See my general comment - this needs to be changed.

File: test/jdk/java/awt/print/bug8023392/bug8023392.java [hunk 2]
```diff
@@ -46,9 +46,10 @@
 
 public class bug8023392 extends Applet {
     static final String[] instructions = {
+        "Please select variable radiobutton in applet.",
```
Comment (prrace) [inline test/jdk/java/awt/print/bug8023392/bug8023392.java:49]: why?  And the text reads oddly,  I think you mean
"Please select the Radio Button for applet size labeled "sariable"  in the test harness window"
but still, why ??

Comment (prsadhuk) [reply test/jdk/java/awt/print/bug8023392/bug8023392.java:49]: Even now, in "fixed" mode the "Print" button is not visible and we need to select "Variable" to see the button. I just made it more official by changing the instruction.
BTW, I dont see the general comment about the label...I will work on the getting the inset than adding spaces in label.

**Review by prrace — APPROVED** [review]

**Pull request closed without merging**